src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml

```diff
@@ -31,6 +31,10 @@
                 <Setter Property="Foreground" Value="{DynamicResource {x:Static utilities:CodeAnalysisColors.CheckBoxTextBrushKey}}"/>
             </Style>
 
+            <Style TargetType="TextBlock">
+                <Setter Property="Foreground" Value="{StaticResource ForegroundText}"/>
```

Comment:
Ah, I see now that this is what everything else *already* did, you just extracted it to the style to make it less error-prone in the future.